assets/dev/js/editor/components/icons-manager/classes/icon-library.js

```diff
@@ -0,0 +1,152 @@
+import Store from './store';
+
+const IconLibrary = class {
+	loaded = {};
+
+	notifyCallback = null;
+
+	enqueueCSS = ( url ) => {
+		return new Promise( function( resolve ) {
+			if ( ! document.querySelector( 'link[href="' + url + '"]' ) ) {
```

Comment:
We prefer to use `jQuery` for every elements and DOM manipulations

Reply:
replaced with elementor.helpers.enqueuePreviewStylesheet()